app/models/user.rb

```diff
@@ -1,7 +1,10 @@
 class User < ApplicationRecord
-  validates :name, presence: true
-  validates :bio, presence: true
-  validates :posts_counter, numericality: { only_integer: true, greated_than_or_equal_to: 0 }
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] The application does not ask for confirmation of email, it is required that the application asks for confirmation of email after sign up and the user has to confirm their emails (in the terminal) before taking them to the home page. Kindly add `:confirmable` here so that devise knows about the confirmable we want to have in the application.